src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -226,6 +226,18 @@ typedef struct otSysInfraNetIfAddressCounters
  */
 void otSysCountInfraNetifAddresses(otSysInfraNetIfAddressCounters *aAddressCounters);
 
+/**
+ * Sets the infrastructure network interface and the ICMPv6 socket.
+ *
+ * This function specifies the network interface name and the ICMPv6 socket on that interface. Then we can call
+ * otBorderRoutingInit() to let Border Routing work on that interface.
```

Comment:
> Then we call

Clarify "we" in this. In particular, clarify whether the caller needs to call `otBorderRoutingInit()`.